streams/src/test/java/org/apache/kafka/streams/integration/utils/KafkaEmbedded.java

```diff
@@ -104,23 +110,14 @@ private Properties effectiveConfigFrom(final Properties initialConfig) {
      */
     @SuppressWarnings("WeakerAccess")
     public String brokerList() {
-        final EndPoint endPoint = kafka.advertisedListeners().head();
+        final EndPoint endPoint = kafka.config().effectiveAdvertisedListeners().head();
         return endPoint.host() + ":" + endPoint.port();
     }
 
-
-    /**
-     * The ZooKeeper connection string aka `zookeeper.connect`.
-     */
-    @SuppressWarnings("WeakerAccess")
-    public String zookeeperConnect() {
-        return effectiveConfig.getProperty("zookeeper.connect", DEFAULT_ZK_CONNECT);
```

Comment:
Can we also remove `ZkSessionTimeoutMsProp` `KafkaEmbedded.effectiveConfigFrom` line 99 as we don't need it anymore.